tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>Minor typo in the last paragraph ("fre" should be three).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>